inlong-manager/manager-test/src/main/resources/h2/apache_inlong_manager.sql

```diff
@@ -791,5 +791,29 @@ VALUES ('audit_sdk_collect', 'SDK', 0, '1'),
        ('audit_sort_kudu_output', 'KUDU', 1, '26');
 
 -- ----------------------------
+-- Table structure for inlong_tenant
+-- ----------------------------
+CREATE TABLE IF NOT EXISTS `inlong_tenant`
```

Comment:
Please add this table into the `manager-test` module.

Reply:
I have added this table into the sql in h2 dictionary.